src/GeneralStateTestsFiller/stExample/paygasRevertFiller.json

```diff
@@ -22,11 +22,15 @@
                 "network" : [">=Istanbul"],
                 "result" : {
                     "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
-                        "balance" : "1000000000000000000",
+                        "balance" : "999999999999755820",
+                        "nonce" : "0",
                         "storage" : {
                             "0x00" : "0x00",
                             "0xff" : "0xff"
                         }
+                    },
+                    "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
+                        "balance" : "244180"
```

Comment:
Why do we need a new assert on the coinbase's balance?

Reply:
Well, given that the whole gas payment logic has been changed for AA, I just wanted to include an assert that checks that gas payment reached the miner successfully. Already helped me find a bug where
`pre_contract_balance != post_contract_balance + miner_balance`

Reply:
Didn't mean to imply this was an issue! I was just curious if something changed that needed asserting.